plugins/powerpaste.md

```diff
@@ -295,3 +295,39 @@ tinymce.init({
 The PowerPaste plugin provides the following JavaScript command.
 
 {% include commands/powerpaste-cmds.md %}
+
+{% include misc/support-powerpaste.md %}
+
+## Common questions and troubleshooting PowerPaste behavior
+
+### What happens when copy and pasting from Microsoft Word?
+
+When content is copied from an application (such as Microsoft Word), the application places an HTML representation of the copied content onto the computer's clipboard. PowerPaste uses the HTML from the clipboard and cannot access the file directly.
+
+> **Note**: Web browsers and the applications running in them cannot directly access files on the computer for security reasons.
+
+Microsoft Word or Microsoft Excel can create content that does not have equivalent HTML. The HTML provided to the clipboard by the application is the application's "best effort" at representing the content as HTML. Depending on the complexity of the source document, the content pasted into {{site.productname}} using PowerPaste may not be an exact representation of what the content looked like in the original application.
+
+### Why are some images or elements from Microsoft Word not appearing?
+
+Some "images" in Microsoft Word cannot be represented as image files in a HTML document, such as: charts, drawings, and "Word Art". PowerPaste may not be able to paste these items into the {{site.productname}} editor, because they were not represented as HTML-compatible images on the clipboard.
+
+Microsoft Word can also create content that cannot be accurately recreated in HTML, such as columns, page headers and page footers. Some of these elements may not be copied to the clipboard by Microsoft Word, such as page headers and footers.
+
+### How can I see what is on the clipboard?
+
+To view the HTML of content pasted from the clipboard:
+
+* If you are using Microsoft Internet Explorer 11, visit: [{{site.companyname}} Clipboard Viewer for Microsoft Internet Explorer 11](http://static.ephox.com/clipboard/clipboardtestie11.html).
+* If you are using any other browser, visit: [{{site.companyname}} Clipboard Viewer](http://static.ephox.com/clipboard/clipboardtest.html).
+
+### Why would Microsoft Internet Explorer 11 show different results from every other supported browser?
+
+Microsoft Internet Explorer interacts with Microsoft Word content differently than all other browsers. When pasting, Microsoft Internet Explorer transforms and cleans up Microsoft Word content before pasting it into {{site.productname}}. This behavior only occurs when Microsoft content is pasted into Microsoft Internet Explorer and cannot be disabled.
```

Comment:
```suggestion
Microsoft Internet Explorer interacts with Microsoft Word content differently than all other browsers. When pasting, Microsoft Internet Explorer transforms and cleans up Microsoft Word content before pasting it into {{site.productname}}. {{site.productname}} also receives different data from the clipboard on Microsoft Internet Explorer than with any other browser. This behavior only occurs when Microsoft content is pasted into Microsoft Internet Explorer and cannot be disabled.
```
maybe?

Reply:
We possibly shouldn't say `TinyMCE also receives` here, as it's not specific to TinyMCE. It's what IE 11 is providing on the clipboard.

Reply:
May have been fixed by https://github.com/tinymce/tinymce-docs/pull/1752/commits/625721ea21fa6a12a9776de96efe4f98a913bbd8